browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -727,10 +727,20 @@ index 0000000000000000000000000000000000000000..587287d52fde2735cbae34a27a0f673b
 +    ]
 +}
 diff --git a/Source/JavaScriptCore/inspector/protocol/Network.json b/Source/JavaScriptCore/inspector/protocol/Network.json
-index 65ab2092b0ffd0ead3da1ddccd398d4f4179f51a..2d9ef40a20df819193c9a5867fbf6f8f8bf46b48 100644
+index 65ab2092b0ffd0ead3da1ddccd398d4f4179f51a..84762978c93fbe18c104f8c626672588ee053e36 100644
 --- a/Source/JavaScriptCore/inspector/protocol/Network.json
 +++ b/Source/JavaScriptCore/inspector/protocol/Network.json
-@@ -321,6 +321,13 @@
+@@ -77,7 +77,8 @@
+                 { "name": "source", "type": "string", "enum": ["unknown", "network", "memory-cache", "disk-cache", "service-worker", "inspector-override"], "description": "Specifies where the response came from." },
+                 { "name": "requestHeaders", "$ref": "Headers", "optional": true, "description": "Refined HTTP request headers that were actually transmitted over the network." },
+                 { "name": "timing", "$ref": "ResourceTiming", "optional": true, "description": "Timing information for the given request." },
+-                { "name": "security", "$ref": "Security.Security", "optional": true, "description": "The security information for the given request." }
++                { "name": "security", "$ref": "Security.Security", "optional": true, "description": "The security information for the given request." },
++                { "name": "address", "type": "string", "optional": true, "description": "Remote IP address and port." }
```

Comment:
If it's convenient, lets return `serverIPaddress::string` and `port::number` in distinct fields to match CR and FF.

Reply:
It's not convenient!